db/controllers/controller.js

```diff
@@ -0,0 +1,25 @@
+const db = require("../connection");
+
+const fetchTopicsFromDatabase = async () => {
```

Comment:
This is a model and should live in the models file and should definitely be moved into that file